Gemfile.lock

```diff
@@ -456,7 +456,7 @@ GEM
       money (~> 6.12)
     money (6.14.0)
       i18n (>= 0.6.4, <= 2)
-    msgpack (1.4.1)
+    msgpack (1.4.2)
```

Comment:
I just happened to experience an issue with this gem. Bundler complains that it can't find 1.4.1 anywhere and updating it to 1.4.2 fixed it.